synapse/server.py

```diff
@@ -238,6 +241,11 @@ def __init__(self, hostname: str, config: HomeServerConfig, reactor=None, **kwar
         for depname in kwargs:
             setattr(self, depname, kwargs[depname])
 
+    def get_instance_id(self):
+        """A unique ID for this synapse process instance.
```

Comment:
wouldn't mind some words about what this is used for. "This is used to distinguish instances in worker-based deployments"?